### PR TITLE
root: 6.09.02 -> 6.10.00

### PR DESCRIPTION
**Pull request by veprbl**
also workaround #26197

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/science/misc/root/default.nix [hunk 1]
```diff
@@ -38,6 +34,7 @@ stdenv.mkDerivation rec {
     "-Drpath=ON"
     "-DCMAKE_INSTALL_LIBDIR=lib"
     "-DCMAKE_INSTALL_INCLUDEDIR=include"
+    "-DOPENGL_INCLUDE_DIR=${OpenGL}/Library/Frameworks"
```
Comment (Mic92) [inline pkgs/applications/science/misc/root/default.nix]: is it intended to this by default on all platforms?

**Comment by Mic92** [comment]
does not build in a sandbox, because it downloads TBB while installing::

```
Scanning dependencies of target TBB            
[  0%] Creating directories for 'TBB'          
[  0%] Performing download step (download, verify and extract) for 'TBB'
```

**Comment by veprbl** [comment]
I've disabled implicit multithreading which required Threading Building Blocks library. This is now tested on sandbox in NixOS and on OSX.

**Comment by veprbl** [comment]
@Mic92 Thank you!

**Pull request merged**